.github/actions/compile-commit/action.yml

```diff
@@ -22,10 +22,12 @@ runs:
         MAVEN_INSTALL_OPTS="-Xmx3G -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
         MAVEN_COMPILE_COMMITS="-B --strict-checksums --quiet -T C1 -DskipTests -Dmaven.source.skip=true -Dair.check.skip-all=true -Dmaven.javadoc.skip=true --no-snapshot-updates --no-transfer-progress -pl !:trino-server-rpm"
         MAVEN_GIB="-P gib -Dgib.referenceBranch=refs/remotes/origin/${{ inputs.base_ref }}"
+        RETRY: .github/bin/retry
         # -------
 
         # For building with Maven we need MAVEN_OPTS to equal MAVEN_INSTALL_OPTS
         export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+        $RETRY $MAVEN dependency:go-offline
```

Comment:
This unfortunately downloads the entire world (i.e. all plugins referred to in release + all dependencies of all modules).
However it shouldn't be problematic since `./.github/actions/setup` will hopefully have restored a cache already.